modules/quant_purger/quant_purger.install

```diff
@@ -34,3 +34,32 @@ function quant_purger_schema() {
   ];
   return $schema;
 }
+
+/**
+ * Implements hook_install().
+ */
+function quant_purger_install() {
+  // Disable the plugin to prevent it from getting in a way of uninstall.
+  $plugins = \Drupal::service('purge.queuers')->getPluginsEnabled();
+  array_push($plugins, 'quant');
+
+  // Set enabled plugins, including quant.
+  \Drupal::service('purge.queuers')->setPluginsEnabled($plugins);
```

Comment:
The plugin isn't added automatically when the module is enabled? I'm confused why the hook_install is here though the hook_install makes sense to prevent the uninstall error.

Reply:
@kepol yeah this one is a bit funny. In the scenario when you Uninstall and then Install it again, the plugin won't be enabled because Uninstall specifically saves config in `purge.queuers` that this plugin is disabled.

It feels a bit hacky but I couldn't come up with a better way to overcome the issue.